### PR TITLE
TOOLS-2451 thoth logs is broken

### PR DESCRIPTION
**Pull request by jlevon**

### CODE REVIEW

**Comment by bahamat** [comment]
Is this going to make all old logs inaccessible?

**Comment by jlevon** [comment]
Are we running thoth downrev enough to not have:

commit e3604dc177989e5e51551168b87d96f98be4053f
Author:     Bryan Cantrill <bcantrill@acm.org>
AuthorDate: Wed Apr 29 10:21:22 2015 -0700

    joyent/manta-thoth#105 sdc-thoth doesn't include the year in log file path

?

**Comment by bahamat** [comment]
Oh, we’re way past that.

File: bin/thoth [hunk 1]
```diff
@@ -3427,7 +3427,7 @@ handlers.logs = function (client, argv)
 	var timePath = function (hours) {
 		var t = new Date(new Date().valueOf() - (3600 * hours * 1000));
 
-		return (sprintf('%02d/%02d/%02d',
+		return (sprintf('%04d/%02d/%02d/%02d', t.getFullYear(),
```
Comment (bahamat) [inline bin/thoth]: `getUTCFullYear()`?

**Comment by jlevon** [comment]
I have a lot more coming, so this can wait I think.

**Pull request merged**